### ISSUE TITLE
How to disable "[ble: press RET to continue]" prompt?

### ISSUE THREAD

**Issue by adoyle-h** [akinomyoga/ble.sh#304]
I'm using https://github.com/lincheney/fzf-tab-completion . When pressed `ls<Tab>`, the ble.sh prints `[ble: press RET to continue]`.
I want to disable this prompt.

<img width="335" alt="CleanShot 2023-03-16 at 21 52 56@2x" src="https://user-images.githubusercontent.com/1998490/225638730-22446762-2f4c-4bd3-bd66-10d8b153e13e.png">

**Comment by akinomyoga** [comment]
> I'm using https://github.com/lincheney/fzf-tab-completion . When pressed `ls<Tab>`,

I tried that setting with `bind -x '"\t": fzf_bash_completion'` but cannot reproduce the problem. How did you set up `fzf-tab-completion`?

> the ble.sh prints `[ble: press RET to continue]`

This message is supposed to be only printed when the redirection of `stderr` is failing due to some setup issues or due to the crash of ble.sh. Since your setup in #303 seems to be some problems with the redirections of `stderr` and `stdout`, I suspect that affects the behavior of the fzf-tab-completion binding. How do you source ble.sh currently?

**Comment by adoyle-h** [comment]
## fzf-tab-completion

Load below content with `1> >(...)` and `2> >(...)` redirections mentioned in #303 .

```
source fzf-tab-completion/bash/fzf-bash-completion.sh

bind -x '"\t": fzf_bash_completion'
_fzf_bash_completion_loading_msg() {
  printf '%b%s \n' "${PS1@P}" "${READLINE_LINE}"  | tail -n 1
}
```

## ble.sh

```
source ble/share/blesh/ble.sh &> "$(tty)"
```

**Comment by akinomyoga** [comment]
> ```bash
> _fzf_bash_completion_loading_msg() {
>   printf '%b%s \n' "${PS1@P}" "${READLINE_LINE}"  | tail -n 1
> }
> ```

Ah, so you are outputting `PS1`. The variable `PS1` is set to a different value `[ble: press RET to continue]` while the user inputs are processed. So far, I haven't think users might want to access `PS1` through keybindings.

If you just want to disable it, you can remove the above `"${PS1@P}"` (and the corresponding `%b` in the format string).

In commit 2eadcd5b, I changed `PS1` to be restored to the user's value while running `bind -x` commands.

**Comment by adoyle-h** [comment]
It get fixed in 2eadcd5. I appreciate your help.

I need to use `PS1` because fzf_bash_completion erase the cursorline to print `_fzf_bash_completion_loading_msg`. So I need to refill the `PS1` text.

**Comment by akinomyoga** [comment]
Thanks for checking!
